src/main/ui/src/components/ParkView/ParkInfo.js

```diff
@@ -0,0 +1,152 @@
+import React, {Component} from 'react';
+import './ParkInfo.css';
+import {Container} from 'react-bootstrap';
+import {Header} from '../Header/Header.jsx';
+import Carousel from 'react-bootstrap/Carousel';
+import useState from 'react';
+import Button from 'react-bootstrap/Button';
+import AddNote from '../Notes/AddNote.jsx';
+
+
+
+
+class ParkInfo extends Component {
```

Comment:
we are using a different format for component declaration.

https://github.com/Jul-2022-LC-LiftOff/Parks-N-Rec/blob/master/src/main/ui/src/components/SearchByName/SearchByName.jsx

checkout how it's done here. 

Also the file extension should be `.jsx`